index.bs

```diff
@@ -4747,22 +4757,24 @@ a numbered step. If outdented, it (today) is rendered as a bullet in the midst o
-    outputs=] in the <code>[=authdataextensions|extensions=]</code> in |authData| are as expected, considering the [=client
-    extension input=] values that were given in <code>|options|.{{PublicKeyCredentialRequestOptions/extensions}}</code>
-    and any specific policy of the [=[RP]=] regarding unsolicited extensions, i.e., those that were not specified as part of
-    <code>|options|.{{PublicKeyCredentialRequestOptions/extensions}}</code>.
-    In the general case, the meaning of "are as expected" is specific to the [=[RP]=] and which extensions are in use.
-
-    Note:  [=Client platforms=] MAY enact local policy that sets additional [=authenticator extensions=] or
-    [=client extensions=] and thus cause values to appear in the [=authenticator extension outputs=] or
-    [=client extension outputs=] that were not originally specified as part of
-    <code>|options|.{{PublicKeyCredentialRequestOptions/extensions}}</code>. [=[RPS]=] MUST be prepared to handle such
-    situations, whether it be to ignore the unsolicited extensions or reject the assertion. The [=[RP]=] can make this
-    decision based on local policy and the extensions in use.
-
-    Note: Since all extensions are OPTIONAL for both the [=client=] and the [=authenticator=], the [=[RP]=] MUST also be
-    prepared to handle cases where none or not all of the requested extensions were acted upon.
+    <li id='authn-ceremony-verify-extension-outputs'>
+        Verify that the values of the [=client extension outputs=] in |clientExtensionResults| and the [=authenticator extension
+        outputs=] in the <code>[=authdataextensions|extensions=]</code> in |authData| are as expected, considering the [=client
+        extension input=] values that were given in <code>|options|.{{PublicKeyCredentialRequestOptions/extensions}}</code>
+        and any specific policy of the [=[RP]=] regarding unsolicited extensions, i.e., those that were not specified as part of
+        <code>|options|.{{PublicKeyCredentialRequestOptions/extensions}}</code>.
+        In the general case, the meaning of "are as expected" is specific to the [=[RP]=] and which extensions are in use.
+
+        Note:  [=Client platforms=] MAY enact local policy that sets additional [=authenticator extensions=] or
+        [=client extensions=] and thus cause values to appear in the [=authenticator extension outputs=] or
+        [=client extension outputs=] that were not originally specified as part of
+        <code>|options|.{{PublicKeyCredentialRequestOptions/extensions}}</code>. [=[RPS]=] MUST be prepared to handle such
+        situations, whether it be to ignore the unsolicited extensions or reject the assertion. The [=[RP]=] can make this
+        decision based on local policy and the extensions in use.
+
+        Note: Since all extensions are OPTIONAL for both the [=client=] and the [=authenticator=], the [=[RP]=] MUST also be
```

Comment:
I think we need to be more specific how the attestation conveyance "direct" would be handled for durable credentials when DPK is NOT supported. See line 1907 above.  The current description seems to indicate the AAGUID and attestation would relate to the DPK only - and not to the durable credential.

Reply:
I can see having separate AAGUID for the DPK and public key credential source.    For the direct attestation that might be self with a AAGUID that generally describes the sync fabric, iCloud etc, where the DPK AAGUID would be specific to the hardware authenticator and describe how the DPK was stored.   It might be possible to get a sync fabric certified at L1 as a software authenticator where the DPK might have a higher level.

Reply:
Wouldn't we have to redefine what AAGUID means then? It is to identity an Authenticator today, not a key.

Reply:
In a way though, if the private key is synced between devices, then some kind of sync mechanism must exist as the "backing" to that key. So having a unique AAGUID for that sync mechanism is good, because in a way it *is* identifying the authenticator as "a sync mechanism of type X".  Saying this AAGUID's are already a little fuzzy WRT to key storage since a yubikey (for example) will use the same AAGUID for a discoverable key as a key-wrapped-key. 

An argument exists though that the seperate AAGUID isn't needed because say an AAGUID of an iphone SE, combined with the backup state flags / dpk, indicates the hardware involved precisely, and that the private key uses the apple sync backend. So you don't really need a seperate AAGUID to identify what sync mechanism is used, you only need the set of AAGUID's of hardware that can participate in that sync mechanism.

Reply:
> Wouldn't we have to redefine what AAGUID means then? It is to identify an Authenticator today, not a key.

Today, the (private) keys don't leave the authenticator.  As a result, for each key you know what authenticator model (identified by the AAGUID) exclusively manages/protects the key.

This is true for keys than cannot be restored to different authenticator models (incl. the DPK in most if not all cases).

For multi-device keys, the AAGUID would then denote the "Multi-device fabric" - not a distinct HW component.

To my knowledge, we never specified the authenticator to be a distinct HW component.  
So we *could* treat the "multi-device fabric" as an authenticator and assign an AAGUID to a specific one (at some point).